cmd/utils/utils.go

```diff
@@ -0,0 +1,47 @@
+package utils
```

Comment:
Calling this `utils` is an anti-pattern since it could become a a dumping ground for random stuff. It seems to make sense to have this in the `analytics` package since it deals with analytics-specific functionality.